gateway/bin/docker-start.sh

```diff
@@ -2,7 +2,7 @@
 
 mkdir -p /usr/src/app/gateway.level /usr/src/app/transactions.level \
     /usr/src/app/db
-chown -R hummingbot:hummingbot /usr/src/app/logs /usr/src/app/conf \
+chown -R hummingbot:hummingbot /usr/src/app/logs /usr/src/app/conf /usr/src/app/certs \
```

Comment:
Some folders are created here in the Docker start script, rather than in the Dockerfile. I think that is because they were added later, not because they necessarily should here.

Would it make sense to consolidates all the `mkdir` and `chown` steps in this file to the Gateway Dockerfile instead?

Reply:
Totally agree, I removed this file completely